marlin/src/parameters.rs

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU General Public License
 // along with the snarkVM library. If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{MarlinTestnet1, ProvingKey, VerifyingKey, SRS};
+use crate::{MarlinTestnet1, ProvingKey, VerifyingKey, URS};
```

Comment:
Note: we use keep the name SRS, since URS often means uniform random string, which is a string that has no structured. Marlin still uses SRS, not an URS.

Reply:
Will clean up, had name collision with `SRS` struct

Reply:
`UnivesalSRS` can be a good name.

Reply:
Solved in #301.